src/patternfly/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,189 @@
+@import "../../patternfly-utilities";
+
+.pf-c-tabs-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+// Tabs
+.pf-c-tabs {
+  /* stylelint-disable */
+
+  // Primary Tab Items
+  --pf-c-tabs__item--BackgroundColor:       var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs__item--current--Color:        var(--pf-global--active-color--100);
```

Comment:
switching this to `--pf-global--primary-color--100` fixes the theme issue but I think we need to investigate this further.